### PR TITLE
Remove thumbnail expand icon on `PostListing`'s `postTitleLine`

### PR DESCRIPTION
**Pull request by alectrocute**
Hi Lemdevs!

In this PR:

- Remove extra expand icon on `PostListing`'s `postTitleLine`
- Why would a user want to expand a thumbnail? It's a confusing button.

Screenshot 2023-06-26 at 2 33 39 PM

Thanks!

### CODE REVIEW

**Comment by jsit** [comment]
> Why would a user want to expand a thumbnail? It's a confusing button.

Why wouldn't they? Isn't this only for image posts? Could be nice, rather than navigating to the domain where the image is hosted.

**Comment by dessalines** [comment]
I agree, image expanding isn't really necessary for link posts. But it is for image posts, as that's essentially one of the best utilities of RES.

File: src/shared/components/post/post-listing.tsx [hunk 1]
```diff
@@ -482,26 +482,13 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
               this.postLink
             )}
           </h5>
-          {(url && isImage(url)) ||
-            (post.thumbnail_url && (
```
Comment (dessalines) [inline src/shared/components/post/post-listing.tsx:486]: I think all that's necessary is to remove the or condition. We do want to show an expando for image posts, just not for link posts.

**Comment by jsit** [comment]
Should this be handled in #1596? I sense a conflict coming.

**Comment by alectrocute** [comment]
I thought clicking the thumbnail *is* the expando for image posts?

**Comment by dessalines** [comment]
> I thought clicking the thumbnail is the expando for image posts?

Oh yeah I forgot. In that case yep it makes sense to remove this. That'll help @jsit 's PR as it removes the conflicting icons.

**Pull request merged**